firmware/thruna/FSM/Inc/Io/Io_AcceleratorPedals.h

```diff
@@ -2,6 +2,11 @@
 
 #include <stdbool.h>
 
+/**
+ * @return returns absolute angle based on pedal position using cosine law
+ */
+float calcAppsAngle(float cos_law_coefficent, float cos_law_denominator, float pot_len);
+
```

Comment:
It's best practice to only add the functions that you need in other files to the header. This keeps the header file clean (you only expose the stuff that other files care about, so its easier to read) and avoids name conflicts (if defined non-statically in the header file, no other function can have this name).

Reply:
Ahh, I see good to know